packages/web-frontend/public/index.html

```diff
@@ -45,26 +45,18 @@
     </script>
     <!-- End Google Analytics -->
 
-    <!-- Hotjar Tracking Code for https://tupaia.org/ -->
-    <script>
-      (function(h, o, t, j, a, r) {
-        h.hj =
-          h.hj ||
-          function() {
-            (h.hj.q = h.hj.q || []).push(arguments);
-          };
-        h._hjSettings = { hjid: 1665619, hjsv: 6 };
-        a = o.getElementsByTagName('head')[0];
-        r = o.createElement('script');
-        r.async = 1;
-        r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
-        a.appendChild(r);
-      })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
+    <!-- Microsoft Clarity Tracking Code for https://tupaia.org -->
+    <script type="text/javascript">
+      (function(c,l,a,r,i,t,y){
+        c[a]=c[a]||function(){(c[a].q=c[a].q||[]).push(arguments)};
+        t=l.createElement(r);t.async=1;t.src="https://www.clarity.ms/tag/"+i;
+        y=l.getElementsByTagName(r)[0];y.parentNode.insertBefore(t,y);
+      })(window, document, "clarity", "script", "89ft6ytrun");
```

Comment:
Is it possible in the clarity interface to filter out non-prod usage? If not, could we conditionally include this? It's the same for google analytics actually...it's quite tricky in the interface to only include "real" usage